### PR TITLE
Issue 6938: Document how to enable DurableDataLogs via CLI that are not impacted by data corruption

### LINKED ISSUE #6938: Document how to enable DurableDataLogs via CLI that are not impacted by data corruption
**Description of the document**
In some cases, a `DurableDataLog` (i.e., `BookkeeperLog`) may be disabled by some failure that does not necessarily involve data corruption. For instance, we have seen this happening if a Segment Store runs out of direct memory. In these cases, once the memory pressure is solved, the logs can be safely enabled. We need a document explaining the steps to perform such an operation.

### PR DESCRIPTION
**Pull request by RaulGracia**
**Change log description**  
Added a new document in Recovery Procedures section of documentation to safely enable Bookkeeper logs.

**Purpose of the change**  
Fixes #6938.

**What the code does**  
New document added to the Recovery Procedures.

**How to verify it**  
No code changes.

### CODE REVIEW

**Review by abhinb — APPROVED** [review]

**Review by kuldeepk3 — APPROVED** [review]
Looks good to me.

File: documentation/src/docs/recovery-procedures/enable-durable-data-log.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+This command lists all the available Bookkeeper Logs in this cluster (please, be sure that the Pravega
+Admin CLI has the `pravegaservice.container.count` parameter set to the same number of Segment Containers 
+as in the cluster itself). The output of the above command shows that all the disabled Bookkeeper logs
+exhibit `"enabled": false`.
+
+3. Next, we need to be sure that we can recover the disabled Bookkeeper log(s) as a way to verify that
+there are no data corruption issues. To this end, you need to run the following Pravega Admin CLI command
+on all the disabled logs:
+```
+container recover [ID_OF_DISABLED_CONTAINER]
```
Comment (ShwethaSNayak) [inline documentation/src/docs/recovery-procedures/enable-durable-data-log.md:73]: This allows to check whether container can recover from Persistent errors, but if BookKeeper log been disabled due to transient errors (like OOM), then should we provide some way to detect such errors before enabling Bookkeeper log?

**Review by ShwethaSNayak — APPROVED** [review]
Other than above query documentation looks good to me

File: documentation/src/docs/recovery-procedures/enable-durable-data-log.md (hunk 1 above)
Comment (RaulGracia) [reply documentation/src/docs/recovery-procedures/enable-durable-data-log.md:73]: This is the point of running `container recover`: if it works, data and metadata is good, so we can safely enable the container. If it does not work, then there is a data corruption issue that should be handled differently.

**Pull request merged**